frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -49,6 +49,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                     <NavDropdown.Item href="/admin/createcommons">Create Commons</NavDropdown.Item>
                     <NavDropdown.Item href="/admin/users">Users</NavDropdown.Item>
                     <NavDropdown.Item href="/admin/listcommons">List Commons</NavDropdown.Item>
+                    <NavDropdown.Item href="/admin/showleaderboard">Show Leaderboard</NavDropdown.Item>
```

Comment:
Instead of different menu item here, it should probably be a button next to the Edit and Delete buttons on the list of commons that is found from the `List Commons` options.

That's because a leaderboard is going to be specific to a commons.